docs/interceptors.md

```diff
@@ -257,6 +257,55 @@ For more information around adding changed files, see the following examples
 - [github-add-changed-files-pr](../examples/v1beta1/github-add-changed-files-pr)
 - [github-add-changed-files-push-cel](../examples/v1beta1/github-add-changed-files-push-cel)
 
+#### Owners validation for pull requests
+
+The GitHub `Interceptor` supports the ability to halt processing on a pull request if the user is not listed in the [owners](https://www.kubernetes.dev/docs/guide/owners/) file or is not a repository/organization member/owner. This feature can be used to prevent unnecessary execution of a PipelineRun or TaskRun. The GitHub `Interceptor` also supports the ability to trigger a PipelineRun/TaskRun through a comment that contains `/ok-to-test` on a pull request by an owner. 
+
+This feature will also work against private GitHub repositories by supplying a [GitHub personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) in the `personalAccessToken` field.
+
+> NOTE: Owners validation requires (at a minimum) the `pull_request` and `issue_comment` GitHub event types.
```

Comment:
minor: not sure I understand the "at a minimum" part - pull_request and issue_comment are the only two valid event types right?

Reply:
For Github Interceptor there are other valid event types like `push`. My intention here is to say that for the Owners validation `pull_request` and `issue_comment` event types are the minimum requirement.

Reply:
ok, but currently we only support pull_request and issue_comment? there may be others that we support later (like push?)

Reply:
No, for Owners validation these are the only two it would ever support

Reply:
I'd just rephrase this to say Owners validation requires the event type to be pull_request or issue_comment - the "at a minimum" phrase I think is a bit confusing